### PR TITLE
fix duplicate key mapping

### PR DESCRIPTION
**Pull request by ronie**
fix duplicate key mapping (https://github.com/xbmc/xbmc/blob/master/system/keymaps/keyboard.xml#L151) introduced by https://github.com/xhaggi/xbmc/commit/779c7c4c710168a70cd8f25a0e1f380781f2cb7a

@xhaggi

### CODE REVIEW

**Comment by Razzeee** [comment]
Closing bracket also needs the change

File: system/keymaps/keyboard.xml [hunk 1]
```diff
@@ -48,7 +48,7 @@
       <browser_back mod="longpress">ActivateWindow(Home)</browser_back>
       <key id='65446'>Back</key>
       <m>Menu</m>
-      <m mod="ctrl">ActivateWindow(PlayerControls)</m>
+      <n mod="ctrl">ActivateWindow(PlayerControls)</m>
```
Comment (xhaggi) [inline system/keymaps/keyboard.xml]: AS @Razzeee said the closing tag needs also be changed to `</n>` because it's  `</m>` atm.

**Comment by xhaggi** [comment]
Sorry for this mistake and apart from the wrong closing tag I don't have any objections. Thanks @ronie.

**Comment by ronie** [comment]
thanx for the extra pair of eyes :-)

**Pull request merged**